src/express/utils/create-lambda-request-handler.ts

```diff
@@ -41,6 +41,13 @@ export function createLambdaRequestHandler(
           environment,
           event: {
             ...getRequestHeaders(req),
+            requestContext: {
+              protocol: req.protocol,
+              httpMethod: req.method,
+              path: req.path,
+              stage: `$default`,
```

Comment:
Please change this to
```suggestion
              stage: `prod`,
```

Reply:
done